client/components/Navbar.jsx

```diff
@@ -2,11 +2,28 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 const Navbar = () => (
-  <div className="container-fluid border">
-    <div className="row">
-      <Link to="/"><h1 className="col-md-10 col-sm-8 col-xs-6">Ribbit</h1></Link>
-      <Link to="/login"><h2 className="col-md-2 col-sm-4 col-xs-6">signup/login</h2></Link>
-    </div>
+  <div className="container-fluid">
+    <header className="header">
+      <div role="navigation" className="navbar navbar-default navbar-fixed-top">
+        <div className="container">
+          <div className="navbar-header"><Link to="/" className="navbar-brand scroll-to"><img src="../ribbit.png" alt="logo" className="hidden-xs hidden-sm" /><span className="sr-only">Go to homepage</span></Link>
```

Comment:
I'm guessing this threw a linter warning about the line being too long. I might go ahead and break up this line into a vertical flow for readability :)